sechub-pds-solutions/findsecuritybugs/docker/FindSecurityBugs-Debian.dockerfile

```
@@ -7,69 +7,44 @@ FROM ${BASE_IMAGE}
 # The remaining arguments need to be placed after the `FROM`
 # See: https://ryandaniels.ca/blog/docker-dockerfile-arg-from-arg-trouble/
 
-# Arguments
-ARG PDS_FOLDER="/pds"
-ARG SCRIPT_FOLDER="/scripts"
-ARG WORKSPACE="/workspace"
-ARG FSB_VERSION="1.12.0"
-ARG FSB_SHA256SUM="a50bd4741a68c6886bbc03d20da9ded44bce4dd7d0d2eee19ceb338dd644cd55"
-ARG PDS_VERSION="0.31.0"
-
-# Environment
-ENV TOOL_FOLDER="/tools"
-ENV DOWNLOAD_FOLDER="/downloads"
-ENV MOCK_FOLDER="$SCRIPT_FOLDER/mocks"
-ENV PDS_VERSION="${PDS_VERSION}"
-ENV SHARED_VOLUMES="/shared_volumes"
-ENV SHARED_VOLUME_UPLOAD_DIR="$SHARED_VOLUMES/uploads"
+LABEL org.opencontainers.image.source="https://github.com/mercedes-benz/sechub"
+LABEL org.opencontainers.image.title="SecHub FindSecurityBugs+PDS Image"
+LABEL org.opencontainers.image.description="A container which combines FindSecurityBugs with the SecHub Product Delegation Server (PDS)"
+LABEL maintainer="SecHub FOSS Team"
 
-# Create tool, pds, shared volume and download folder
-RUN  mkdir --parents "$TOOL_FOLDER" "$DOWNLOAD_FOLDER" "$PDS_FOLDER" "$SHARED_VOLUME_UPLOAD_DIR" "$WORKSPACE"
+# Arguments
+ARG FINDSECURITYBUGS_VERSION="1.12.0"
+ARG FINDSECURITYBUGS_SHA256SUM="a50bd4741a68c6886bbc03d20da9ded44bce4dd7d0d2eee19ceb338dd644cd55"
 
-# non-root user
-# using fixed group and user ids
-RUN groupadd --gid 2323 pds \
-     && useradd --uid 2323 --no-log-init --create-home --gid pds pds
+# Environment variables in container
+ENV FINDSECURITYBUGS_VERSION="${FINDSECURITYBUGS_VERSION}"
 
-# Update image and install dependencies
-ENV DEBIAN_FRONTEND noninteractive
+USER root
 
-RUN apt-get update && \
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
     apt-get --assume-yes upgrade  && \
     apt-get --assume-yes install dos2unix unzip wget openjdk-17-jre-headless libxml2-utils tree && \
```

We now have two openjdk's installed. I am not sure what will be used for FindSecBugs. On the other hand, we could have two PDS-Base Image builds, one with Java 11 and one with Java 17 (both are LTS versions). 

FindSecBugs depends on Java 17, otherwise it cannot analyze code compiled with Java 17. However, FindBugs running on Java 17 can analyze code compiled with Java 11.

The `tree` command is already included in the base image.
 
```suggestion
    apt-get --assume-yes install dos2unix unzip wget libxml2-utils && \
```

---

Good point.

Since openjdk-17-jre-headless is installed as last java version, it becomes the default via /etc/alternatives/java
```
pds@pds-findsecuritybugs-66fc486486-xlk9n:/workspace$ java -version
openjdk version "17.0.4" 2022-07-19
OpenJDK Runtime Environment (build 17.0.4+8-Debian-1deb11u1)
OpenJDK 64-Bit Server VM (build 17.0.4+8-Debian-1deb11u1, mixed mode, sharing)
```
So we are using Java 17 as desired.

Removed `tree` package here.